composer.json

```diff
@@ -17,12 +17,13 @@
     },
     "require": {
         "php": ">=5.4.0",
-        "jms/serializer": "^0.16 || ^1.0",
-        "ramsey/uuid": "^3.0 || ^4.0",
-        "symfony/http-foundation": "^2.0 || ^3.0 || ^4.0 || ^5.0"
+        "jms/serializer": "^0.16 | ^1 | ^3 | ^4",
+        "ramsey/uuid": "^3 | ^4",
+        "symfony/http-foundation": "^2 | ^3 | ^4 | ^5"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.0.0"
+        "phpunit/phpunit": "^7.5.15",
+        "symfony/mime": "^5"
```

Comment:
Workaround for a problem I've talked about in issue #61. After merging this request I'm going to make PR to solve it.